commcare_connect/opportunity/tasks.py

```diff
@@ -73,3 +81,51 @@ def generate_payment_export(opportunity_id: int, export_format: str):
         content = content.encode()
     default_storage.save(export_tmp_name, ContentFile(content))
     return export_tmp_name
+
+
+@celery_app.task()
+def send_notification_inactive_users():
+    opportunity_accesses = OpportunityAccess.objects.filter(
+        opportunity__active=True,
+        opportunity__end_date__gt=datetime.date.today(),
+    ).select_related("opportunity")
+    messages = []
+    for access in opportunity_accesses:
+        has_claimed_opportunity = OpportunityClaim.objects.filter(opportunity_access=access).exists()
+        if has_claimed_opportunity:
+            message = _get_deliver_message(access)
+        else:
+            message = _get_learn_message(access)
+        if isinstance(message, Message):
```

Comment:
The possible types for message are either `Message` or None so I think it would be clearer to use `if messages` otherwise it gives the impression that there may be other types being returned.

Reply:
Fixed. https://github.com/dimagi/commcare-connect/pull/181/commits/f629e9ed52c71c65c8e04a078a53ad67e6a89441